### PR TITLE
add natsort

### PR DESCRIPTION
**Pull request by bubkoo**

### CODE REVIEW

File: files/natsort/update.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "packageManager": "github",
+  "name": "natsort",
+  "repo": "bubkoo/natsort",
+  "files": {
+    "include": [
+      "index.js",
```
Comment (megawac) [inline files/natsort/update.json]: just include `dist/` please (`"basePath": "dist/"`)

**Comment by bubkoo** [comment]
I have update the config files, everything is ok now?

File: files/natsort/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author = "bubkoo"
+github = "https://github.com/bubkoo/natsort"
+homepage = "https://github.com/bubkoo/natsort"
+description = "Javascript natural sort algorithm with unicode support."
+mainfile = "./dist/natsort.min.js"
```
Comment (jimaek) [inline files/natsort/info.ini]: Just `natsort.min.js` here

File: files/natsort/update.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "packageManager": "github",
+  "name": "natsort",
+  "repo": "bubkoo/natsort",
+  "files": {
+    "include": [
+      "./dist/"
```
Comment (jimaek) [inline files/natsort/update.json]: Just `dist/` here

**Comment by bubkoo** [comment]
@jimaek 
@megawac 
Help me to check the files.

**Comment by bradvogel** [comment]
@jimaek Natsort isn't showing up in http://www.jsdelivr.com/?query=natsort. Has this been published?

**Comment by bubkoo** [comment]
@jimaek 

I have only upload `update.json` and `info.ini`, will the server auto fetch the files specified in the `update.json`?

**Comment by jimaek** [comment]
There are no tags https://github.com/bubkoo/natsort/releases
thats why nothing  happened

**Comment by bubkoo** [comment]
I just create one, how long does this take effect?

**Comment by jimaek** [comment]
It runs every 30 minutes

**Comment by bubkoo** [comment]
Thanks a lot

**Comment by bradvogel** [comment]
Still not showing up - http://www.jsdelivr.com/?query=natsort. Is the tag right?

**Comment by jimaek** [comment]
@MartinKolarik Any idea why its failing now? update.json looks correct

**Comment by MartinKolarik** [comment]
@jimaek existing local branch would be my first guess.

**Comment by jimaek** [comment]
You were right. Any idea how we can avoid this bug? I dont get why it creates a local branch but never pushes it

**Comment by MartinKolarik** [comment]
@jimaek I have no idea why it sometimes fails, but I think the easiest solution would be deleting it if it exists.

**Comment by jimaek** [comment]
maybe the bot can check for local branches automatically and delete them before starting the scan?
I dont think that will break anything

**Comment by MartinKolarik** [comment]
yeah that's what I meant

**Comment by jimaek** [comment]
ok, if its safe then lets do it

**Comment by MartinKolarik** [comment]
jsdelivr/libgrabber#67

**Pull request merged**